README.md

```diff
@@ -30,74 +28,74 @@
 
 ## What is MapReader?
 
-MapReader is an end-to-end computer vision (CV) pipeline for exploring and analyzing images at scale.
+**MapReader is a open-source python library for exploring and analyzing map images at scale.**
 
-<figure align="center">
-  <img src="https://raw.githubusercontent.com/Living-with-machines/MapReader/main/docs/source/_static/river_banner_8bit.png"
-      alt="Annotated Map with Prediction Outputs"
-      width="70%">
-</figure>
+It contains two different pipelines:
 
-MapReader was developed in the [Living with Machines](https://livingwithmachines.ac.uk/) project to analyze large collections of historical maps but is a _**generalizable**_ computer vision pipeline which can be applied to _**any images**_ in a wide variety of domains.
+- Classification pipeline: This pipeline enables users to fine-tune a classification model and predict the labels of patches created from a parent image.
+- Text spotting pipeline: This pipeline enables users to detect and recognize text in map images.
 
-## Overview
+MapReader was developed in the [Living with Machines](https://livingwithmachines.ac.uk/) project to analyze large collections of historical maps but is a _**generalizable**_ computer vision tool which can be applied to _**any images**_ in a wide variety of domains.
 
-MapReader is a groundbreaking interdisciplinary tool that emerged from a specific set of geospatial historical research questions. It was inspired by methods in biomedical imaging and geographic information science, which were adapted for use by historians, for example in our [Journal of Victorian Culture](https://doi.org/10.1093/jvcult/vcab009) and [Geospatial Humanities 2022 SIGSPATIAL workshop](https://arxiv.org/abs/2111.15592) papers. The success of the tool subsequently generated interest from plant phenotype researchers working with large image datasets, and so MapReader is an example of cross-pollination between the humanities and the sciences made possible by reproducible data science.
```

Comment:
Where to put this text?

Reply:
Perhaps in 'what is mapreader' 'origin of mapreader' section? And change "MapReader is" to "MapReader was".